imports/ui/views/organizations/AboutUsMission/AboutUsImages.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+
+class AboutUsImages extends Component {
+  render() {
+    return (
+      <div style={styles.images}>
+        {this.props.images.map((image, index) =>
```

Comment:
I feel like that would be weird since the map takes up several lines of code and makes up almost all of the code in the component.